<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Клиент</string>
  </property>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>77</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </widget>
  <widget class="QLabel" name="clientStatusLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>147</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Статус: Ожидание связи с сервером</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>57</y>
     <width>172</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Ip</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="hostLineEdit">
      <property name="placeholderText">
       <string>127.0.0.1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>117</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отправить</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>117</y>
     <width>172</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Порт</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="portLineEdit">
      <property name="placeholderText">
       <string>6666</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Соединение с 
сервером</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
